src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,104 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classnames from 'classnames';
 import './FindMovie.scss';
-
+import PropTypes from 'prop-types';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { getMovie } from '../../api/movies';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ onAdd, error, setError }) => {
+  const [value, setValue] = useState('');
```

Comment:
```suggestion
  const [query, setQuery] = useState('');
```
Value is not a meaningful name. It's hard to understand, for what purposes you use it. Better to give more meaningful names